inception-imls-external/src/test/java/de/tudarmstadt/ukp/inception/recommendation/imls/external/RemoteStringMatchingNerRecommender.java

```diff
@@ -72,7 +73,7 @@ public RemoteStringMatchingNerRecommender(Recommender aRecommender)
         featureName = aRecommender.getFeature().getName();
     }
 
-    public void train(String aTrainingRequestJson) throws Exception
+    public void train(String aTrainingRequestJson) throws UIMAException, SAXException, IOException, RecommendationException
```

Comment:
This line is very long. Please use checkstyle (see [here](https://inception-project.github.io//releases/0.12.1/docs/developer-guide.html#_setting_up_checkstyle_and_formatting) to identify and fix these kind of issues.